code/modules/awaymissions/mission_code/Cabin.dm

```diff
@@ -103,6 +103,24 @@
 		..()
 		new L.plank_type(src.loc, 1 + round(potency / 25))
 
+/obj/structure/ladder/unbreakable/rune
+	name = "\improper Teleportation Rune"
+	desc = "Could lead anywhere."
+	icon = 'icons/obj/rune.dmi'
+	icon_state = "1"
+	color = rgb(0,0,255)
+
+/obj/structure/ladder/unbreakable/rune/ComponentInitialize()
+	. = ..()
+	AddElement(/datum/element/update_icon_blocker)
```

Comment:
can you put this on Initialize() instead? We're trying to move away from ComponentInitialize from what I understand.

Reply:
Yes it's deprecated it shouldn't be used

Reply:
Resolved